libs/MVS/SceneDensify.cpp

```diff
@@ -1737,37 +1737,42 @@ bool Scene::DenseReconstruction()
 		data.progress.Release();
 	}
 
-	// fuse all depth-maps
-	pointcloud.Release();
-	data.detphMaps.FuseDepthMaps(pointcloud, OPTDENSE::nEstimateColors == 2, OPTDENSE::nEstimateNormals == 2);
-	#if TD_VERBOSE != TD_VERBOSE_OFF
-	if (g_nVerbosityLevel > 2) {
-		// print number of points with 3+ views
-		size_t nPoints1m(0), nPoints2(0), nPoints3p(0);
-		FOREACHPTR(pViews, pointcloud.pointViews) {
-			switch (pViews->GetSize())
-			{
-			case 0:
-			case 1:
-				++nPoints1m;
-				break;
-			case 2:
-				++nPoints2;
-				break;
-			default:
-				++nPoints3p;
+	// fuse all depth-maps if needed
+	if(OPTDENSE::bFuse)
```

Comment:
I think it will look more clean if let the code as it was, and just add `return true` after if

Reply:
Thanks, agree!

Reply:
@fixed via 186e8f1b40ef118eb54f528da1c63a66f394a09a